lib/credit_card_validations/plugins/diners_us.rb

```diff
@@ -0,0 +1,8 @@
+# :diners_us:
+#   :rules:
+#     - :length:
+#       - 16
+#       :prefixes:
+#       - '54'
+#       - '55'
+CreditCardValidations.add_brand(:diners_us, {length: 16, prefixes: %w(54 55)}, {brand_name: 'Diners Club US'})
```

Comment:
Line is too long. [110/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.